### DISCUSSION TITLE
Can we get a public macOSicons.com API?

### DISCUSSION THREAD

**Discussion by insomnia-creator** [elrumo/macOS_Big_Sur_icons_replacements#1375] [Polls]
The title says it all, various endpoints for interacting with macOSicons.com via REST preferably.

**Comment by daniel4-scratch** [comment]
Yes
![8B417EF2-C8C9-48FC-ACD4-185A4B66B9BD](https://user-images.githubusercontent.com/65277548/165741662-3575f72a-ed38-4a08-b364-3876417584de.jpeg)
